framework/queue/Job.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\queue;
+
+use yii\base\Object;
+
+/**
+ * Job is the base class for queue job classes.
+ *
+ * @author Alexander Kochetov <creocoder@gmail.com>
+ * @since 2.0.5
+ */
+abstract class Job extends Object
```

Comment:
Why this is needed?
Job itself should not be exposed at public interface. If particular queue implemetation requires extra class for the Job, it can add it and use internally.